docs/src/languages/java.md

```diff
@@ -88,15 +103,16 @@ If you have issues with either of these plugins, please open issues on their res
 
 ## Example Configs
 
-### Zed Java Classpath
+### Zed Java Settings
 
-You can optionally configure the class path that JDTLS uses with:
+You can optionally configure the version of JDTLS to download (see <#optional-install-jdtls>) or the class path that JDTLS uses with:
```

Comment:
The `<#optional-install-jdtls>` part probably needs to fix, but I'm not sure how. Hopefully you get the idea here.